.gitlab/machines.yml

```diff
@@ -10,6 +10,7 @@
     PARTITION: pdebug
     BUILD_ALLOC: srun -N 1 -c 36 -p pdebug -t 60
     TEST_ALLOC: ''
+    LC_MODULES: ""
```

Comment:
The changes in this file can be removed as they were a redundant fix to the CUDA c++17 issue.

Reply:
Done in latest commit.